### ISSUE TITLE
Can't pip install v4

### ISSUE THREAD

**Issue by wmcguinn-octave** [innobi/pantab#319]
**Describe the bug**
On my Mac M1, pip cannot find any version above 3.0.2. I've tried on python 3.10, 3.11, and 3.12

**To Reproduce**

```shell
pip install pantab==4.1.0
```

```
ERROR: Could not find a version that satisfies the requirement pantab==4.1.0 (from versions: 0.0.1.dev1, 0.0.1.dev2, 0.0.1.dev3, 0.0.1a1, 0.0.1a2, 0.0.1a3, 0.0.1b1, 0.0.1b2, 0.0.1b3, 0.0.1b4, 0.0.1b5, 0.1.0, 0.1.1, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 1.0.0, 1.0.1, 1.1.0, 1.1.1, 2.0.0, 2.1.0, 2.1.1, 3.0.0, 3.0.1, 3.0.2)
ERROR: No matching distribution found for pantab==4.1.0
```

**Expected behavior**
Package installs successfully

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS 14.5 on Apple M1 Pro chip

**Additional context**

**Comment by WillAyd** [comment]
v4 was released before the Tableau Hyper API supported arm chips on macOS. The next release will have this feature (you are welcome to install directly from github and try)

**Comment by wmcguinn-octave** [comment]
@WillAyd Thank you. Installing from the source worked for me.

**Comment by WillAyd** [comment]
Just released pantab 5.0.0 today which supports Apple Silicon - should be good here now. Thanks for the issue!
